app-games/src/main/java/com/aptoide/android/aptoidegames/LaunchIntentExtensions.kt

```diff
@@ -22,6 +22,10 @@ val Intent?.isAhab: Boolean get() = this?.getBooleanExtra(AHAB_NOTIFICATION, fal
 
 fun Intent.markAsAhab(): Intent = putExtra(AHAB_NOTIFICATION, hasDeepLink)
 
+//External url
+const val EXTERNAL_KEY = "external_url"
+const val WEBVIEW_KEY = "webview"
```

Comment:
This key is not described in the ticket. So please add the missing decision there if any or remove it if this is not a requirement.